src/parser/paladin/protection/modules/spells/WordOfGlory.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent, HealEvent, ApplyBuffEvent, RemoveBuffEvent } from 'parser/core/Events';
+import { When, ThresholdStyle } from 'parser/core/ParseResults';
+import { Options } from 'parser/core/Module';
+import HIT_TYPES from 'game/HIT_TYPES';
+
+const OVERHEAL_THRESHOLD = 0.75;
+
+export default class WordOfGlory extends Analyzer {
+  _sl = {
+    count: 0,
+    wasted: 0,
+    // sotr casts w/ SL up.
+    sotrs: 0,
+  };
+
+  _casts = {
+    free: 0,
+    payed: 0,
+    // casts that overhealed by enough to be concerning
+    overhealed: 0,
+  };
+
+
+  constructor(options: Options) {
+    super(options);
+
+    this.addEventListener(Events.cast.spell(SPELLS.WORD_OF_GLORY).by(SELECTED_PLAYER), this._cast);
+    this.addEventListener(Events.cast.spell(SPELLS.SHIELD_OF_THE_RIGHTEOUS).by(SELECTED_PLAYER), this._sotrCast);
+    this.addEventListener(Events.applybuff.spell(SPELLS.SHINING_LIGHT), this._slApply);
+    this.addEventListener(Events.removebuff.spell(SPELLS.SHINING_LIGHT), this._slRemove);
+    this.addEventListener(Events.heal.spell(SPELLS.WORD_OF_GLORY).by(SELECTED_PLAYER), this._heal);
+  }
+
+  private _heal(event: HealEvent) {
```

Comment:
No need for _ prefix with the private keyword. That was just a thing for JS where there was no such thing

Reply:
I still find it useful as an indicator when using the method, but I'm ambivalent on it and could remove the _s.